exampleReactNative/src/tests/TestSuite_Integration.ts

```diff
@@ -181,29 +181,35 @@ export class TestSuite_Integration extends TestSuite {
 
     async testTestUserAgents() {
 
+        let tempMtoken: WultraMobileToken
```

Comment:
This is a "workaround" to mimic the native behavior of hiding the `userAgent` property from all services.

Reply:
ok